.github/workflows/sync.yml

```diff
@@ -0,0 +1,27 @@
+name: Repo Sync
+on:
+  push:
+    branches: [main]
+    paths:
+      - .github/actions/issue-in-project
+      - .github/workflows/boards.yml
+      - .github/workflows/issues.yml
+env:
+  DRY_RUN: false
+jobs:
+  Github_Workflow_Sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: varunsridharan/action-github-workflow-sync@3.1
```

Comment:
I've had more success with https://github.com/BetaHuhn/repo-file-sync-action in the past, could we use that instead? The issues I saw were mostly related to breaking changes in varunsridharan's action.

Reply:
No problem I just tried to pick an action that had some number of starts, I can def take a look at this one instead!

Reply:
Should be almost identical in variables!